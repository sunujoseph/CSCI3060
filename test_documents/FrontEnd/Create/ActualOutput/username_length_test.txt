path recieved: C:\Users\pwner\Documents\Uoit\Year 5 Semester 2\CSCI 3060U\Project\CSCI3060\test_documents\FrontEnd\Create\ActualDTF\username_length_test.txt
Enter Username: temp: |
create
usÍÍÍÍÍÍÍııııİİİz{óÊx|
username: |admin1
create
u| not found
Error: Not Logged In
Error: Not Logged In
