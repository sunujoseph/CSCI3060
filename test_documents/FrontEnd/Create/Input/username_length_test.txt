login
admin1
create
user1234567890123456789
