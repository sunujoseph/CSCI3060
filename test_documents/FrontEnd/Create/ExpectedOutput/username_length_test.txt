Enter Username: 
Enter Username For New User: 
Error: Invalid Name
Enter Username For New User: 